packages/fields-unsplash/src/Implementation.js

```diff
@@ -256,6 +256,12 @@ export class Unsplash extends Implementation {
   gqlCreateInputFields() {
     return [`${this.path}: String`];
   }
+  getBackingTypes() {
+    // FIXME
+    // @mitchell can you help here?
+    // { type: Object }
```

Comment:
When you say

```
{ type: Object }
```

Aren't the keys Unsplash supports known?

If you do want just a plain object though, it's usually

```
{ type: Record<string, unknown> }
```

There's also a `JSONValue` type in `keystone-next/types` which is more what you're after (i.e json keys can't be functions or dates, so it limits values to valid json) but it's worth checking with @mitchellhamilton how we'd approach ensuring that is imported into the file that is generated with this info and how you'd specify that here.

Reply:
I'd just make it `Record<string, any>` for now